beets/autotag/hooks.py

```diff
@@ -609,25 +619,17 @@ def album_candidates(items, artist, album, va_likely, extra_tags):
     constrain the search.
     """
 
+    common_args = [album, len(items), extra_tags]
     # Base candidates if we have album and artist to match.
     if artist and album:
```

Comment:
This stanza and the next could share a single `if config["musicbrainz"]["enabled"]:`.